sdk/storage/azblob/zc_storage_error_test.go

```diff
@@ -38,3 +40,38 @@ func (s *azblobUnrecordedTestSuite) TestErrorResponseUnmarshal() {
 		})
 	}
 }
+
+func (s *azblobUnrecordedTestSuite) TestInternalServerErrorWrapping() {
+	t := s.T()
+
+	cases := []struct {
+		name   string
+		err    *InternalError
+		target error
+		result bool
+	}{
+		{
+			"errors.Is reports true if checked against an internal error",
```

Comment:
This was the existing behavior, just captured in a test now.

I'm not sure if this behavior is actually desirable but I don't know what the compatibility concerns may be.